src/utilities/indexedDB-helper.js

```diff
@@ -29,12 +32,15 @@ export const getDb = async () =>
       console.log("DB ON UPDATE NEEDED");
       const db = event.target.result;
       try {
-        db.deleteObjectStore(TABLE_NAME);
+        db.deleteObjectStore(SELECTION_TABLE_NAME);
+        db.deleteObjectStore(DATA_TABLE_NAME);
       } catch {}
-      db.createObjectStore(TABLE_NAME, { keyPath: "selectionId" });
+      db.createObjectStore(SELECTION_TABLE_NAME, { keyPath: "selectionId" });
+      db.createObjectStore(DATA_TABLE_NAME, { keyPath: "selectionId" });
```

Comment:
since 'selection' is not relevant to 'data' this should probably be 'dataId' instead of 'selectionId'